tests/functional/test_package.py

```diff
@@ -69,10 +69,57 @@ def test_package_installation():
         else:
             TestRun.executor.run_expect_success("rpm -e iotrace")
 
-    with TestRun.step("Check if io trace was uninstalled"):
+    with TestRun.step("Check if iotrace was uninstalled"):
         TestRun.executor.run_expect_fail("iotrace -V")
 
     if iotrace_installed:
         with TestRun.step("Reinstall iotrace"):
             install_iotrace()
 
+
+def test_source_package_installation():
+    TestRun.LOGGER.info("Testing source package installation")
+
+    iotrace: IotracePlugin = TestRun.plugins['iotrace']
+    work_path: str = iotrace.working_dir
+
+    with TestRun.step("Building iotrace source package"):
```

Comment:
Is it local or remote? We should create package locally and upload it on test machine.

Reply:
If we create the source package on the controller there may be an issue with any concurrent instances trying to run this test. The whole repository is copied to dut anyway, so it shouldn't matter where a source package is created - it doesn't actually compile anything, it just tar'ed together.